apps/tools/src/fund.command.ts

```diff
@@ -0,0 +1,183 @@
+import { DisbursementSummary, FundingService } from '@app/blockchain/funding.service'
+import { networkConfig, NetworkConfig } from '@app/utils/config/network.config'
+import { Inject } from '@nestjs/common'
+import BigNumber from 'bignumber.js'
+import commander from 'commander'
+import { Command, Console, createSpinner } from 'nestjs-console'
+import { Logger } from 'nestjs-pino'
+import { TransactionReceipt } from 'web3-core'
+
+@Console({
+  name: 'fund',
+  description: 'Manage the fund and relayers',
+})
+export class FundCommand {
+  constructor(
+    @Inject(networkConfig.KEY)
+    private readonly networkCfg: NetworkConfig,
+    private readonly fundingSvc: FundingService,
+    private readonly logger: Logger
+  ) {}
+
+  @Command({
+    command: 'disburse',
+    description: 'Disburse funds to the relayers and their meta-tx wallets',
+    options: [
+      {
+        flags: '-co, --celo-amount <celoValue>',
+        required: false,
+        defaultValue: 1,
+        fn: (v) => parseFloat(v),
+        description: 'The amount of Celo to transfer to each relayer'
+      },
+      {
+        flags: '-cu, --cusd-amount <cUSDValue>',
+        required: false,
+        defaultValue: 2,
+        fn: (v) => parseFloat(v),
+        description: 'The amount of cUSD to transfer to each relayer meta-tx wallet'
+      },
+      {
+        flags: '-r, --relayer <relayer>',
+        required: false,
+        defaultValue: [],
+        fn: (r, list) => list.concat([r]),
+        description: 'Which relayer to fund (repetable), if non provided will fund all'
```

Comment:
Is the usage `fund -r 0x1... -r 0x2...`?
```suggestion
        description: 'Which relayer to fund (repeatable), if non provided will fund all'
```